juju/controller.py

```diff
@@ -257,13 +263,20 @@ def logout(self, force=False):
         """
         pass
 
-    def get_model(self, name):
+    async def get_model(self, uuid):
```

Comment:
-1 here. I think Controller.get_model() should return a connected Model instance. Would've mentioned this sooner but I thought this PR was still WIP.